### PR TITLE
RBS-13885: semanticstack-rql workflows

### PR DESCRIPTION
**Pull request by zvirinsky**

### CODE REVIEW

**Comment by zvirinsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by zvirinsky** [comment]
recheck

File: .github/workflows/pull-request-check.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Pull Request Check
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
```
Comment (MelleD) [inline .github/workflows/pull-request-check.yml:4]: Do we need this types or just use the default?

File: .github/workflows/pull-request-check.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          server-id: central
+          cache: 'maven'
+          overwrite-settings: false
+
+      - name: Build and Test
```
Comment (MelleD) [inline .github/workflows/pull-request-check.yml:32]: Maybe we should also check if we can create the Docu with Antora in the PR

**Comment by zvirinsky** [comment]
recheck

**Review by MelleD — APPROVED** [review]

**Pull request merged**